### PR TITLE
Feature/tag selectable

### PR DESCRIPTION
**Pull request by JoanClaret**
Adding tag selectable component

### CODE REVIEW

File: components/tag/selectable/src/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+import Check from '@schibstedspain/sui-svgiconset/lib/Check'
+
+const TagSelectableClassName = ({isSelected}) => cx('sui-TagSelectable', {
```
Comment (kikoruiz) [inline components/tag/selectable/src/index.js]: A nivel de convención, al no ser una clase ni un componente de React, no debería ir en mayúscula.

File: components/tag/selectable/src/index.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  'is-selected': isSelected
+})
+
+const TagSelectable = ({onClick, isSelected, label, icon: Icon = Check} = {}) =>
+  <button
+    onClick={onClick}
+    className={TagSelectableClassName({isSelected: isSelected})}>
```
Comment (kikoruiz) [inline components/tag/selectable/src/index.js]: You can do just `{ isSelected }` ;)

File: components/tag/selectable/src/index.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+import Check from '@schibstedspain/sui-svgiconset/lib/Check'
+
+const TagSelectableClassName = ({isSelected}) => cx('sui-TagSelectable', {
+  'is-selected': isSelected
+})
+
+const TagSelectable = ({onClick, isSelected, label, icon: Icon = Check} = {}) =>
```
Comment (kikoruiz) [inline components/tag/selectable/src/index.js]: Best destructuring object ever seen 👏

File: components/tag/selectable/src/index.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  @include sui-tag;
+
+  &.is-selected {
+    border-color: $c-primary;
+  }
+
+  &-icon {
+    fill: $c-primary !important;
+    margin-right: $m-h-small;
+    width: 14px;
```
Comment (kikoruiz) [inline components/tag/selectable/src/index.scss:14]: We should think of creating a new convention about icon sizes (e.g.: `$size-icon-s, $size-icon-m...`), right now we have a lot of  hardcoded ones.

Comment (JoanClaret) [reply components/tag/selectable/src/index.scss:14]: Absolutely. 
keep in mind that this component will be iterated when we'll implement the motor one.

Comment (kikoruiz) [reply components/tag/selectable/src/index.scss:14]: Sure, I'm just talking about something to be done in a further task.

**Review by kikoruiz — APPROVED** [review]

**Pull request merged**